R/rma-tidiers.R

```diff
@@ -0,0 +1,222 @@
+#' Tidying methods for meta-analyis objects
+#'
+#' These methods tidy the results of meta-analysis objects from the metafor package
+#'
+#' @param x An `rma` created by the `metafor` package.
+#' @inheritParams tidy.lm
+#' @param include_studies Logical. Should individual studies be included in the
+#'    output?
+#' @param ... Additional arguments
+#' @param measure Measure type. See [metafor::rma()]
+#'
+#' @return A `tibble`
```

Comment:
Take a look at https://github.com/tidymodels/broom/blob/master/R/betareg-tidiers.R#L8 for an example of how to document the tidier return. There are three separate functions `return_tidy()`, ..., `return_augment()` to look at. Also I definitely need to document how to use these better.